content/en-US/api/getContextAttributes.md

```diff
@@ -0,0 +1,14 @@
+## Return value
```

Comment:
Can you create a whole page for this method. Run `npm run create -- -i getContextAttributes` and select "Property" from the list.

Reply:
I am not sure if there is enough to create a whole page for this one. I just want to add one more return value, one paragraph. I put alpha and desynchronized to show the location for this paragraph. 
I also searched for alpha and desynchronized, both of them don't have they own page as well.

Reply:
I understand, but MDN is not a blog. It's more like a dictionary. It needs an entry for every interface, constructor, event, event handler, method, and property.

Reply:
I see, I added a new file for this property. Could you review again? Thank you.